common4j/src/main/com/microsoft/identity/common/java/crypto/StorageEncryptionManager.java

```diff
@@ -109,8 +109,7 @@ public byte[] generate() {
     }
 
     @Override
-    @NonNull
-    public byte[] encrypt(@NonNull final byte[] plaintext)
```

Comment:
Why are we removing the NonNull annootations?

Reply:
nonnull has no effect on primitives (byte array). I just removed as indicated by the warning.